src/pages/404/PageNotFound404.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Suggestion: You could hide your navigation bar on 404 page.

![image](https://user-images.githubusercontent.com/91944618/161415493-cfca722a-f598-4a4f-a709-db7097b599c0.png)

Reply:
i gues what i have seen is app have navigationbar on 404 page incase of blocking user to only go to home page